packages/app/src/public-types.ts

```diff
@@ -139,6 +182,66 @@ export interface FirebaseAppSettings {
   automaticDataCollectionEnabled?: boolean;
 }
 
+/**
+ * @public
+ *
+ * Configuration options given to {@link (initializeServerApp:1) | initializeServerApp()}
+ */
+export interface FirebaseServerAppSettings extends FirebaseAppSettings {
+  /**
+   * An optional Auth ID token used to resume a signed in user session from a client
+   * runtime environment.
+   *
+   * If provided, the FirebaseServerApp instance will work to validate the token even before Auth
+   * is initialized. The result of the validation can be queried via by the application by invoking
+   * {@link (FirebaseServerApp.authIdTokenVerified()}. Awaiting the Promise returned by
+   * {@link (FirebaseServerApp.authIdTokenVerified()} is highly recommended if an authIdToken token
+   * is provided.
+   *
+   * Invoking getAuth() with a FirebaseServerApp configured with a validated authIdToken will cause
```

Comment:
Many of the instances of "will" in these comments could be present tense instead, like "validated authIdToken causes an automatic attempt..."

https://docs.google.com/presentation/d/1Ezb4sp4XD6ItHISccnME-EVH_oJj59R5Naq0IskLrAo/edit#slide=id.ge4febe9b09_0_6